sdgym/s3_utils.py

```diff
@@ -0,0 +1,48 @@
+import boto3
```

Comment:
I would rename this module to `s3.py`, as there is no other place where S3 is mentioned, so there is no ambiguity